### PR TITLE
WT-5345 Add dump timestamp to wt verify config options

### PR DESCRIPTION
**Pull request by jeremytay90**

### CODE REVIEW

**Comment by jeremytay90** [comment]
retest this please

**Comment by lukech** [comment]
retest this please

**Comment by luke-pearson** [comment]
Could you comment on the jira with an example of the old output and the new output on the same table?

**Comment by jeremytay90** [comment]
> Could you comment on the jira with an example of the old output and the new output on the same table?

Will do. There's no real difference with old output per-se but rather just adds output.

File: dist/api_data.py [hunk 1]
```diff
@@ -1319,11 +1319,15 @@ def __ge__(self, other):
         Display the contents of in-memory pages as they are verified,
         using the application's message handler, intended for debugging''',
         type='boolean'),
+    Config('dump_timestamps', 'false', r'''
```
Comment (luke-pearson) [inline dist/api_data.py]: Would it be better to call it dump_timepairs?

Comment (sulabhM) [reply dist/api_data.py]: I will go with whatever we call (timestamp + txn-id) in durable history branch.

Comment (luke-pearson) [reply dist/api_data.py]: Yeah timepair is better.

File: dist/api_data.py [hunk 2]
```diff
@@ -1319,11 +1319,15 @@ def __ge__(self, other):
...
+    Config('dump_time_pairs', 'false', r'''
+        Display aggregated time pair ranges as pages are verified,
+        using the application's message handler, intended for debugging''',
+        type='boolean'),
     Config('strict', 'false', r'''
         Treat any verification problem as an error; by default, verify will
         warn, but not fail, in the case of errors that won't affect future
         behavior (for example, a leaked block)''',
-        type='boolean')
+        type='boolean'),
```
Comment (luke-pearson) [inline dist/api_data.py]: Remove added comma

Comment (luke-pearson) [reply dist/api_data.py]: Never mind, this file generally calls for a comma here.

File: src/btree/bt_vrfy.c [hunk 3]
```diff
@@ -360,16 +367,25 @@ __verify_tree(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *addr_unpack
...
-          __wt_page_type_string(page->type)));
+    /* Optionally dump the address/timestamp information. */
+    if (vs->dump_address || vs->dump_time_pairs) {
+        WT_RET(__wt_msg(session, "%s", __wt_page_type_string(page->type)));
+        if (vs->dump_address)
+            WT_RET(__wt_msg(session, "\t%s", __wt_page_addr_string(session, ref, vs->tmp1)));
+        if (vs->dump_time_pairs) {
+            addr = ref->addr;
+            if (addr != NULL)
+                WT_RET(__wt_msg(session, "\t<%lx:%lx, %lx:%lx>", addr->oldest_start_ts,
```
Comment (luke-pearson) [inline src/btree/bt_vrfy.c]: @sulabhM you may be interested in this. We're printing timestamps as Hex which I think makes more sense.

Comment (agorrod) [reply src/btree/bt_vrfy.c]: @luke-pearson This isn't aimed at me - I think it's important that we print timestamps in the same format that they are printed from MongoDB. Let me know if you need help finding an example of how that is done.

File: src/btree/bt_vrfy.c [hunk 4]
```diff
@@ -360,16 +367,25 @@ __verify_tree(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *addr_unpack
 
     bm = S2BT(session)->bm;
     page = ref->page;
+    addr = NULL;
```
Comment (sulabhM) [inline src/btree/bt_vrfy.c]: variable declarations and assignments should be in the same order. So ```addr``` before ```bm```

File: src/btree/bt_vrfy.c (hunk 3 above)
Comment (sulabhM) [reply src/btree/bt_vrfy.c]: I second @agorrod , it will be best to use ```__wt_timestamp_to_string()``` which prints in MongoDB ways.

Comment (jeremytay90) [reply src/btree/bt_vrfy.c]: In keeping with the current debug printing format in `__debug_cell`, I've changed it to ts/txn,ts/txn

**Comment by jeremytay90** [comment]
retest this please

**Pull request merged**